Solutions/Azure Active Directory/Analytic Rules/Sign-in Burst from Multiple Locations.yaml

```diff
@@ -22,19 +22,22 @@ relevantTechniques:
 query: |
   let aadFunc = (tableName:string){
   table(tableName)
-  | where AppDisplayName == "GitHub.com"
+  | where AppDisplayName =~ "GitHub.com"
   | where ResultType == 0
-  | summarize CountOfLocations = dcount(Location), Locations = make_set(Location), BurstStartTime = min(TimeGenerated), BurstEndTime = max(TimeGenerated) by UserPrincipalName, Type
+  | summarize CountOfLocations = dcount(Location), Locations = make_set(Location,100), BurstStartTime = min(TimeGenerated), BurstEndTime = max(TimeGenerated) by UserPrincipalName, Type
   | where CountOfLocations > 1
```

Comment:
This is quite simplistic, I think we would get better results from using series_decompose_anomalies. Often a user has more than 1 location legitimately especially when locations are close together. series_decompose_anomalies would show us when its out of normal activity for that user.
I think there was a similar query to this where you were going to use series_decompose_anomalies - so this might be a duplicate.